README.en.md

```diff
@@ -0,0 +1,179 @@
+* Use the tccli cvm DescribeInstances command to view which cloud servers are currently on the account.
+* View the list of CBS disks by using the tccli cbs DescribeDisks command.
+
+Take the example of creating a cvm (**Please note that the non-simple type of parameters in the demo must be in the standard json format**):
+```bash
+tccli cvm RunInstances --InstanceChargeType POSTPAID_BY_HOUR --InstanceChargePrepaid '{"Period":1,"RenewFlag":"DISABLE_NOTIFY_AND_MANUAL_RENEW"}'
+ --Placement '{"Zone":"ap-guangzhou-2"}' --InstanceType S1.SMALL1 --ImageId img-8toqc6s3 --SystemDisk '{"DiskType":"CLOUD_BASIC", "DiskSize":50}'
+--InternetAccessible '{"InternetChargeType":"TRAFFIC_POSTPAID_BY_HOUR","InternetMaxBandwidthOut":10,"PublicIpAssigned":true}' --InstanceCount 1
+--InstanceName TCCLI-TEST --LoginSettings '{"Password":"isd@cloud"}' --SecurityGroupIds '["sg-0rszg2vb"]' --HostName TCCLI-HOST-NAME1
+```
+For more features, you can check out the supported products with tccli help and check the supported interfaces with tccli cvm help (example cvm). Use the tccli cbs DescribeDisks help (using the DescribeDisks interface of the cbs product as an example) to view the parameters supported by the interface.
+
+# Advanced Features
+## Multi-version interface access
+Some products may have multiple versions of the interface, and TCCLI accesses the latest version of the interface by default. If you want to access a specific old version of the interface, you can do so in the following way (in cvm example).
+```bash
+# Set the default version of the cvm product: 2017-03-12.
+tccli configure set cvm.version 2017-03-12
+
+# Specify the version number when using it in real time.
+tccli cvm help --version 2017-03-12
+tccli cvm DescribeZones help --version 2017-03-12
+tccli cvm DescribeZones --version 2017-03-12
+```
+## Specify the nearest access point (Endpoint)
+By default, TCCLI requests the nearest interface point to access the service. You can also specify your own Endpoint for a product (using cvm as an example).
+```bash
+# Set the default endpoint of the cvm product.
+tccli configure set cvm.endpoint cvm.ap-guangzhou.tencentcloudapi.com
+
+# Specify in real time when called.
+tccli cvm DescribeZones --endpoint cvm.ap-guangzhou.tencentcloudapi.com
+```
+## Return result filtering
+1. Output without any filtering (take the return of the cvm DescribeZones interface as an example).
+```bash
+[root@VM_180_248_centos ~]# tccli cvm DescribeZones
+{
+    "TotalCount": 4,
+    "ZoneSet": [
+        {
+            "ZoneState": "AVAILABLE",
+            "ZoneId": "100001",
+            "Zone": "ap-guangzhou-1",
+            "ZoneName": "Guangzhou District"
```

Comment:
ZoneName may not be translated because currently API response default setting is zh_CN